eogrow/pipelines/export_maps.py

```diff
@@ -113,102 +168,35 @@ def _get_output_folder(self) -> str:
         _, feature_name = self.config.feature
         return fs.path.join(self.storage.get_folder(self.config.output_folder_key), feature_name)
 
-    def create_maps(self, feature_name: str, patch_names: List[str]) -> None:
-        """A method which creates a joined GeoTIFF maps from a list of exported GeoTIFFs"""
-        if not patch_names:
-            raise ValueError("Cannot create map with an empty list of EOPatch names")
-
-        folder = self._get_output_folder()
-        crs_eopatch_dict = self.eopatch_manager.split_by_utm(patch_names)
-
-        # TODO: This could be parallelized per-crs
-        for crs, eopatch_list in crs_eopatch_dict.items():
-            subfolder = f"UTM_{crs.epsg}"
-            map_name = self.config.map_name or f"{feature_name}.tiff"
-
-            # manually make subfolder, otherwise things fail on S3 in later steps
-            self.storage.filesystem.makedirs(fs.path.join(folder, subfolder), recreate=True)
-            self._create_single_map(folder, subfolder, map_name, eopatch_list)
-
-    def _create_single_map(self, folder: str, subfolder: str, map_name: str, eopatch_list: List[str]) -> None:
-        """Creates a single map
-
-        If necessary, copies the geotiffs to local storage. After the merging transfers the merged map to the
-        appropriate location and removes the per-eopatch tiffs.
-        """
-        geotiff_names = [self.get_geotiff_name(name) for name in eopatch_list]
-        geotiff_paths = [fs.path.join(folder, name) for name in geotiff_names]
-        merged_map_path = fs.path.join(folder, subfolder, map_name)
-
-        temp_fs, sys_paths, sys_map_path = self._prepare_files(map_name, geotiff_names, geotiff_paths, merged_map_path)
-
-        self._merge_maps(sys_paths, sys_map_path)
-
-        if temp_fs is not None:  # copy to storage if a temporary FS was used
-            fs.copy.copy_file(temp_fs, map_name, self.storage.filesystem, merged_map_path)
-            temp_fs.close()
-
-        LOGGER.info("Merged map is saved at %s", get_full_path(self.storage.filesystem, merged_map_path))
-
-        # clean up eopatch tiffs
-        parallelize(
-            self.storage.filesystem.remove, geotiff_paths, workers=None, multiprocess=False, desc="Remove eopatch tiffs"
-        )
-
-    def _prepare_files(
-        self,
-        map_name: str,
-        geotiff_names: List[str],
-        geotiff_paths: List[str],
-        merged_map_path: str,
-    ) -> Tuple[Optional[TempFS], List[str], str]:
-        """Returns system paths that can be used to merge maps.
+    def _prepare_files(self, geotiff_paths: List[str], output_file: str) -> Tuple[Optional[TempFS], List[str], str]:
+        """Returns system paths of geotiffs and output file that can be used to merge maps.
 
         If required files are copied locally and a temporary filesystem object is returned.
         """
         make_local_copies = self.storage.is_on_aws() or self.config.force_local_copies
```

Comment:
when is it beneficial if you're not on AWS and want to use `force_local_copies` ?

Reply:
I'm not sure, in the beginning there was just `force_local_copies`, but then it didnt work on aws without it anymore so I think we added the first part as a hotfix? It was also relevant when we still had mounted S3 filesystems. But yeah, perhaps it could be just automated with the aws check :thinking:

Reply:
I would prefer that, it also decreases the amount of parameters in the config file.

Reply:
Will be dealt with in a separate MR